extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -46,13 +47,14 @@ const TAG = 'amp-pan-zoom';
 const DEFAULT_MAX_SCALE = 3;
 const MAX_ANIMATION_DURATION = 250;
 
-const ELIGIBLE_TAGS = {
-  'svg': true,
-  'DIV': true,
-  'AMP-IMG': true,
-  'AMP-LAYOUT': true,
-  'AMP-SELECTOR': true,
-};
+const ELIGIBLE_TAGS = new Set([
+  'svg',
```

Comment:
Do you happen to know why 'svg' is the only lowercase here? I recognize it was like that beforehand

Reply:
https://www.w3.org/TR/REC-DOM-Level-1/level-one-core.html#ID-745549614